tests/system/matcher/property.py

```diff
@@ -43,3 +44,52 @@ def describe(self):
 
 def prop(path, matcher):
     return Prop(path, matcher)
+
+
+def has_value(name, matcher):
```

Comment:
Please start adding unit tests for this 👼

Reply:
I thought about unit tests but chose not to because these methods are tested implicitly everywhere. I will add docs in the next diff.

> It looks OK, but I have a feeling it gets less expressive

I feel the opposite. It gets way more expressive since it states what we assert.

Reply:
See #6382 for the docstrings.

Reply:
> I feel the opposite. It gets way more expressive since it states what we assert.

I agree, but you are effectively introducing your DSL, which means that a new person should have a clean documentation on how to use it ;)